lib/gui/components/update-notifier/update-notifier.js

```diff
@@ -29,7 +29,18 @@ const UpdateNotifier = angular.module(MODULE_NAME, [
   require('../../os/open-external/open-external')
 ]);
 
-UpdateNotifier.constant('UPDATE_NOTIFIER_SLEEP_TIME', 7 * 24 * 60 * 60 * 100);
+/**
+ * @summary The update notifier sleep time
```

Comment:
Great catch! Done! I also realized we should move the days to milliseconds conversion to `units.js`.